backupd/src/storage/sqlite_db/model.rs

```diff
@@ -0,0 +1,10 @@
+use crate::storage::sqlite_db::schema::files;
+
+#[derive(Queryable, Insertable, Identifiable)]
+#[table_name="files"]
+#[primary_key(real_filename)]
+pub struct DbFile {
+    pub real_filename: String,
+    pub local_filename: String,
```

Comment:
And this should just be `filename`

Reply:
So:
```
id: String,
filename: String,
last_updated: i64,
```

Perhaps we could make id a 128-bit binary field in the database for more "proper" storage of the UUID. Also I might change `last_updated` to say `last_modified` as well.

Reply:
I'd prefer to use a string instead of binary so it easily matches filename. `last_modified` sounds good